### DISCUSSION TITLE
Hint about SyncVars being reset in OnDestroy

### DISCUSSION THREAD

**Discussion by maxkratt** [FirstGearGames/FishNet#832] [Documentation]
It has been requested that the docs mention that SyncVar values will be reset when OnDestroy is called. Perhaps using them in OnDestroy could also give a warning message?

**Comment by FirstGearGames** [comment]
If youd like to add this in the docs please feel free.

There is also this planned for V5 which may be of use -- ideally this will invoke before resets https://trello.com/c/Qou0VDW9/20-ondespawnclient

I'm thinking we can do the same for the server.
